aptos-move/aptos-vm/src/delta_state_view.rs

```diff
@@ -2,22 +2,23 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use anyhow::Result;
+use aptos_aggregator::transaction::ChangeSetExt;
 use aptos_state_view::{StateViewId, TStateView};
 use aptos_types::{
     state_store::{
         state_key::StateKey, state_storage_usage::StateStorageUsage, state_value::StateValue,
     },
-    write_set::{TransactionWrite, WriteSet},
+    write_set::TransactionWrite,
 };
 
 pub struct DeltaStateView<'a, 'b, S> {
     base: &'a S,
-    write_set: &'b WriteSet,
+    change_set: &'b ChangeSetExt,
```

Comment:
FYI, `ChangeSetExt` will be gone soon (If I get some reviews🙃) see here: https://github.com/aptos-labs/aptos-core/pull/8195.

Reply:
Currently, `success_transaction_cleanup` still takes `ChangeSetExt` as input. Shall we wait to land this PR until your PR is landed?

Reply:
No no, land this first, it's just that you are aware that in future we will remove `TransactionOutputExt` and `ChangeSetExt` :)